server/index.js

```diff
@@ -47,6 +47,33 @@ app.get('/api/rooms/:listingID/bookingInfo', async (req, res) => {
   }
 });
 
+/* ------------------------------------------------------------
+New API Routes
+-------------------------------------------------------------*/
+app.post('/api/rooms/:listingID/:bookingDate', (req, res) => {
+  const mySql = getConnection();
```

Comment:
were these connections related to postgres? May rename them if they are to avoid confusion.

Reply:
these were just the new CRUD connections that I created as the first step in SDC. not entirely sure they'll all even be hooked up.